app/models/manageiq/providers/automation_manager.rb

```diff
@@ -5,6 +5,8 @@ class ManageIQ::Providers::AutomationManager < ::ExtManagementSystem
   has_many :configured_systems,           :dependent => :destroy, :foreign_key => "manager_id"
   has_many :configuration_profiles,       :dependent => :destroy, :foreign_key => "manager_id"
   has_many :configuration_scripts,        :dependent => :destroy, :foreign_key => "manager_id"
+  has_many :credentials,                  :class_name => "ManageIQ::Providers::AutomationManager::Authentication",
```

Comment:
Tbh, I think `ManageIQ::Providers::AutomationManager::Credential`  less confusing, because it does not conflate the term `Authentication` with those connected to a Manager via `has_many :authentications` - that is those `Authentications` that are used via `Endpoints` to connect to the manager.

It makes it clear, hey here is another concept.

Plus it is more aligned with the association name.